File: temp/checker.py
'''
    Instructions:
        1. Write your code below "WRITE YOUR LOGIC HERE" after removing the pass statement
        2. Take input from terminal (seperated by spaces) to receive the list 
           e.g. 
           1 0 1 1 0
        3. Return the maximum number of consecutive ones in the list after replacing at max one 0 with 1
'''

def findMaxConsecutiveOnes(nums: list[int]) -> int:
    # WRITE YOUR LOGIC HERE
    count=0
    num_1=1
    num_0=0
    i=0
    while i<len(nums):
        if nums[i]==num_1 and nums[i+1]==num_0 and nums[i+2]==num_1 and nums[i+3]==num_1:
            count+=4
        elif nums[i]==num_1 and nums[i+1]==num_0 and nums[i+2]==num_1:
            count+=3
        i+=1
        return count 
# DO NOT CHANGE ANYTHING BELOW THIS LINE
if __name__ == '__main__':
    nums = list(map(int, input().split()))
    print(findMaxConsecutiveOnes(nums))

File: temp/checker.js
/*
    Instructions:
        1. Write your code below the "WRITE YOUR LOGIC HERE" comment
        2. Take input from terminal (seperated by spaces) to receive the list 
           e.g.
           1 0 1 1 0
        3. Return the maximum number of consecutive ones in the list after replacing at max one 0 with 1
*/

// const { count } = require("console");

var findMaxConsecutiveOnes = function(nums) {
    //WRITE YOUR LOGIC HERE
  let  arr=[];
    let n = parseInt(prompt("Enter array length"));
    for(let i = 0; i<n; i++){
    let userInput = parseInt(prompt("enter an element"));
       arr.push(userInput);
    }
     i = 0;
     count = 0;
     let j = 1;
    while(i<n) {
        if (arr[i]===0);{
         arr.splice(arr[i] , 1);
          i = n;
        }
        i++;
    }
    i = 1 
  while(i<n) {
    if (arr[i]!==0){
        count++;
    }
    i++;
       }
       console.log(count);


 
    
    
 };

/*DO NOT CHANGE ANYTHING BELOW THIS LINE*/
// var readline = require('readline');
// var rl = readline.createInterface({
//     input: process.stdin,
//     output: process.stdout
// });
// rl.question("", function(s) {
//     var nums = s.split(" ").map(Number);
//     console.log(findMaxConsecutiveOnes(nums));
//     rl.close();
// });